core/src/main/java/org/apache/calcite/util/format/FormatElementEnum.java

```diff
@@ -326,7 +377,16 @@ public enum FormatElementEnum implements FormatElement {
       sb.append(String.format(Locale.ROOT, "%03d", calendar.get(Calendar.MILLISECOND)));
     }
   },
-  SS("s", "The second as a decimal number (00-60)") {
+  MCS("", "The millisecond as a decimal number (000000-999999)") {
+    @Override public void format(StringBuilder sb, Date date) {
+      final Calendar calendar = Work.get().calendar;
+      calendar.setTime(date);
+      // It exceeds the precision of Calendar, we fill it with 0.
```

Comment:
Milliseconds are usually between 0 and 999.

Reply:
This is an incorrect function description, it means microsecond here, I'll fix it